inginious/frontend/installer.py

```diff
@@ -550,7 +551,14 @@ def configure_authentication(self, database):
 
         username = self._ask_with_default("Enter the login of the superadmin", "superadmin")
         realname = self._ask_with_default("Enter the name of the superadmin", "INGInious SuperAdmin")
-        email = self._ask_with_default("Enter the email address of the superadmin", "superadmin@inginious.org")
+        while True:
```

Comment:
email = None
      while not email:
         ...  
will require one line less and be more pleasant to read.

Reply:
Fixed in 3da81fd3f7509961d623351c5002dc8a9f583053.